### ISSUE TITLE
[Website] Might want to move or delete "Why is Storyboarder free" insults

### ISSUE THREAD

**Issue by kryptot7** [wonderunit/storyboarder#768]
I saw that this program got featured on Cartoon Brew, a popular animation news site! Congratulations! :smile:

However, some commenters were perplexed at the rant in the latter part of the Why Is Storyboarder Free? section of the website, which insults several people and companies by name, uses strong profanity directed at them, and encourages piracy of overpriced software. The rest of the website is very good and informative, but I found the insults and language to be weird and unnecessary, and they left a rotten taste in my mouth. It could also stir up ill-will between Wonderunit and other companies, bringing in personal politics. For example, a storyboard artist from Sony might want their department to try using Storyboarder instead of another app, but the insults on the site could make the higher-ups bitter against the open-source app. Also, what if a parent or teacher wants to download this for a student? That'd scare them away or at least make them uncomfortable recommending it to kids.

I'm not a professional but I think the insults and ill-will should be deleted from the download site, or at least moved to someone's personal blog as opposed to a company page.

**Comment by automaton-0** [comment]
"Also, what if a parent or teacher wants to download this for a student? That'd scare them away or at least make them uncomfortable recommending it to kids."

Funny. Charles Forman wrote about how this happened, doesn't seem like he's fazed at all.

**Comment by prokoudine** [comment]
Good old Internet police, where would we be without you?

Oh, and "what if a parent or teacher wants to download this for a student. That'd scare them away or at least make them uncomfortable recommending it to kids"? Well, I'm from a project called GIMP. Go on, judge me :)

**Comment by StoryCave** [comment]
That was *THE* hook for me. 

It's unreal how easy people in our biz forget how these huge corporations can sometimes treat their employees with wage-fixing, laying off entire units, dodging the very taxes that would support healthcare while taking government subsidies that could be used to restructure education and employment... I mean, if that's not a middle finger or a dry fist to our humanity, where do we draw the line ?

**Comment by kryptot7** [comment]
True, that's a good point. The website should definitely address that! I
was just saying that I found the "F you"s and insults to be  awkward, not
the underlying values of sticking it to greedy corporations with good, free
softwate. THAT aspect is good. :smile: I'd be happy to see that expressed
in a more constructive way.

On Oct 16, 2017 9:17 AM, "StoryCave" <notifications@github.com> wrote:

That was *THE* hook for me.

It's unreal how easy people in our biz forget how these huge corporations
can sometimes treat their employees with wage-fixing, laying off entire
units, dodging the very taxes that would support healthcare while taking
government subsidies that could be used to restructure education and
employment... I mean, if that's not a middle finger or a dry fist to our
humanity, where do we draw the line ?

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
<https://github.com/wonderunit/storyboarder/issues/768#issuecomment-336883397>,
or mute the thread

**Comment by StoryCave** [comment]
Haha, glad I've added a positive spin to the issue !

In general I agree too that gratuitous F words suck big time, although in that specific context I can't speak for the author, he might have had a different reason and maybe champions this toxic neoliberal paradigm I've just been critical about... if so then fuck him >:D

And now back to drawing.

**Comment by yamlCase** [comment]
I gotta say I was super excited about Storyboarder the more I read down the page... it is a masterpiece in marketing, this is super awesome!.  Then I hit the F bombs.  I'm a grown adult _without kids_ and this made me cringe and second guess everything I read above.  I've been screwed over by enough powerful people and corporations to have the exact same rants so believe me when I say I understand the urge to vomit profanity onto the internet.  But might I suggest doing it in a tweet to at least compartmentalize the product from the personality?
